lib/cc/service/http.rb

```diff
@@ -21,10 +21,10 @@ def service_get(url = nil, body = nil, headers = nil, &block)
     raw_get(url, body, headers, &block)
   end
 
-  def service_post(url, body = nil, headers = nil, &block)
+  def service_post(url, body = nil, headers = nil, response_prefix = nil, &block)
```

Comment:
I don't think this prefixing stuff belongs here in the HTTP layer: it's really a concern you have with presentation of results, not HTTP. It seems like there's another concept hiding here.